packages/modular-scripts/src/utils/getPackageDependencies.ts

```diff
@@ -120,3 +103,46 @@ export async function getPackageDependencies(
     );
   return { manifest, resolutions };
 }
+
+function parseYarnLock(lockFile: string, deps: Dependency): Dependency {
+  return parseYarnLockV1(lockFile, deps) || parseYarnLockV3(lockFile, deps);
```

Comment:
Since we are explicitly catering to yarn 1 and yarn 3, can we add this to `isYarnInstalled` in `startupCheck.ts` ?

Reply:
By this, I mean the check for the version of yarn to be explicit with our compatibility

Reply:
Spoke offline and decided to create a separate ticket to resolve yarn version compatibility